src/lib/containers/oauth/CreateOAuthClient.tsx

```diff
@@ -178,7 +180,7 @@ export const CreateOAuthModal: React.FunctionComponent<
   return (
     <div className="bootstrap-4-backport">
       <Modal
-        show={isShowingModal}
+        show={isShowingModal && !isShowingConfirmModal}
```

Comment:
Hide this modal if the confirm modal is shown. It will instantly reappear when the confirm modal is hidden.